### DISCUSSION TITLE
Resize only one element

### DISCUSSION THREAD

**Discussion by saomartinho** [bvaughn/react-resizable-panels#329] [Q&A] [answered]
Hey, Would like to know if it's possible to use this package just to resize an element? Like, this:

https://github.com/bvaughn/react-resizable-panels/assets/22690341/84c8e169-c375-4131-bddb-b0f283279ec7

Basically, the idea is to create something like a "console," where the dark gray part is on top of the content, and the user can increase/decrease the height of the console without changing the height of the content.

We currently use another package for making these elements resizable, but it has some bugs, and we were considering migrating to react-resizable-panels. We'd like to know if this is possible.

Thanks in advance.

**Comment by bvaughn** [comment]
The video above looks like two panels (top: white, bottom: gray). That seems like a `<PanelGroup direction="vertical">` to me. Am I misunderstanding something?

**Reply by saomartinho** [reply]
Sure, this can be one approach, but the ultimate goal is to keep the height of the light grey component unchanged. Essentially, the darkest grey element is positioned above (fixed bottom position), and we want the user to have the freedom to adjust the height of that element. Makes sense?

**Reply by bvaughn** [reply, marked as answer]
You could accomplish that by setting a fixed height/style to the light gray element and letting its content overflow the panel. That's what I would suggest trying, if you wanted to use this library.

Otherwise I'd say the use case we're describing isn't really one this library supports.

**Reply by saomartinho** [reply]
Thanks, i will try
